Runtime/Game/Requests/AssetRequest.cs

```diff
@@ -23,9 +23,17 @@ public enum AssetFilter
 
 namespace LootLocker.Requests
 {
-    public class LootLockerLinks
+    public class LootLockerLinks : Dictionary<string, string>
     {
-        public string thumbnail { get; set; }
+        public string thumbnail
+        {
+            get
+            {
+                TryGetValue("thumbnail", out var value);
+                return value;
+            }
+            set => thumbnail = value;
```

Comment:
Not familiar enough with C# dictionaries, but will this automatically assign ["thumbnail"] the value so to say? Or should this also have a lambda with TrySetValue?

Reply:
Set most likely doesn't work actually, but to support legacy it needs to still be in there.
thumbnail is most likely null here since the class now is a dictionary, so it will not behave in the same way, which is why we use the TryGetValue() to return "thumbnail" in a dictionary way, so a user will not notice any difference since we're returning the same information, but behind the scenes, nothing is happening.

Reply:
Yes, it works with it, but not without it, unless you revert the change to making it into a dictionary.
So it's kind of a hack, for the user there won't be any difference, but "under the hood" thumbnail is null, which is why we must return it like that. 
Would probably be possible to override the setter instead of the getter, so when setting we could use TrySetValue() essentially, doesn't really make a difference.